src/screens/index.js

```diff
@@ -22,10 +23,35 @@ class App extends PureComponent {
 
   constructor() {
     super();
-
+    this.handleConnectionChangeDebounced = _.debounce(this.handleConnectionChange, 1000);

```

Comment:
Why you can't use this lines (26,27) in the componentDIdMount? 
And if you use constructor it should be
```
 constructor(props) {
    super(props);
```